.github/workflows/linter.yml

```diff
@@ -0,0 +1,14 @@
+name: Lint Check
+
+on: push
```

Comment:
Can we make this run on PRs to `develop` too?

Reply:
sure. and should we restrict the push only to develop?